main.go

```diff
@@ -34,6 +34,7 @@ func init() {
 func main() {
 	var (
 		duration = flag.Duration("duration", time.Hour, "The duration that the credentials will be valid for.")
+		export   = flag.Bool("export", false, "If true, forces the use of bash/shell export statements that can be sourced")
```

Comment:
What do you think about making this a `--format` flag that excepts a string, so you can force bash or powershell style output?

```console
$ assume-role --format bash
$ assume-role --format powershell
```

This should probably also either error out, or print a warning on stderr if `--format` is provided, when a command is also provided.